### PR TITLE
Adding dropwizard-scala-parent-pom

### PR DESCRIPTION
**Pull request by drew222**
Fixing version number, excluding potential jackson conflicts

Adding AWS library

removing unneeded property

test

### CODE REVIEW

File: datateam-parent-pom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.mainstreethub</groupId>
+    <artifactId>dropwizard-scala-parent-pom</artifactId>
+    <version>1.0.2-1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>datateam-parent-pom</artifactId>
```
Comment (handfulofteeth) [inline datateam-parent-pom/pom.xml]: Might want to rename as datateam-dropwizard-scala-parent-pom  (heinously long, I know, but it'd be more accurate)

File: readme.md [hunk 2]
```diff
@@ -55,3 +55,11 @@ child pom.
...
 
 * Upgraded parent pom to ecs-parent-pom version 1.1.0.
+
+## dropwizard-scala-parent-pom
+This pom wraps the dropwizard-parent-pom for development in Scala
+
+### Releases
+
+#### 1.0.2-1
+* Initial release
```
Comment (handfulofteeth) [inline readme.md]: Add comment for datateam pom

File: datateam-parent-pom/pom.xml (hunk 1 above)
Comment (drew222) [inline datateam-parent-pom/pom.xml]: Renamed this.

File: readme.md (hunk 2 above)
Comment (drew222) [inline readme.md]: Added a section for the datateam pom, added a comment / made a new version of the old dropwizard-scala-parent-pom

**Pull request merged**